migration/i8.2z/postgresql/202111020750_IDEMPIERE-5021.sql

```diff
@@ -0,0 +1,687 @@
+;
+
+-- Nov 1, 2021, 4:19:11 PM MYT
+UPDATE AD_Column SET IsUpdateable='N', IsAllowLogging='N', IsAllowCopy='N',Updated=TO_TIMESTAMP('2021-11-01 16:19:11','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=214603
+;
+
+-- Nov 1, 2021, 4:19:43 PM MYT
+UPDATE AD_Column SET IsUpdateable='N', IsAllowLogging='N', IsAllowCopy='N',Updated=TO_TIMESTAMP('2021-11-01 16:19:43','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=214604
+;
+
+-- Nov 1, 2021, 4:19:51 PM MYT
+UPDATE AD_Column SET IsMandatory='Y', IsUpdateable='N', IsAllowLogging='N', IsAllowCopy='N',Updated=TO_TIMESTAMP('2021-11-01 16:19:51','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=214605
+;
+
+-- Nov 1, 2021, 4:20:07 PM MYT
+UPDATE AD_Column SET IsAllowLogging='N', IsAllowCopy='N',Updated=TO_TIMESTAMP('2021-11-01 16:20:07','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=214601
+;
+
+-- Nov 1, 2021, 4:20:14 PM MYT
+UPDATE AD_Column SET IsAllowLogging='N', IsAllowCopy='N',Updated=TO_TIMESTAMP('2021-11-01 16:20:14','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=214602
+;
+
+-- Nov 1, 2021, 4:20:27 PM MYT
+UPDATE AD_Column SET FKConstraintName='ADClient_MStorageReservationLo', FKConstraintType='N',Updated=TO_TIMESTAMP('2021-11-01 16:20:27','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=214597
+;
+
+-- Nov 1, 2021, 4:20:27 PM MYT
+UPDATE AD_Column SET FKConstraintName='ADOrg_MStorageReservationLog', FKConstraintType='N',Updated=TO_TIMESTAMP('2021-11-01 16:20:27','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=214598
+;
+
+-- Nov 1, 2021, 4:20:27 PM MYT
+UPDATE AD_Column SET FKConstraintName='ADTable_MStorageReservationLog', FKConstraintType='N',Updated=TO_TIMESTAMP('2021-11-01 16:20:27','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=214609
+;
+
+-- Nov 1, 2021, 4:20:27 PM MYT
+UPDATE AD_Column SET FKConstraintName='CDocType_MStorageReservationLo', FKConstraintType='N',Updated=TO_TIMESTAMP('2021-11-01 16:20:27','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=214606
+;
+
+-- Nov 1, 2021, 4:20:27 PM MYT
+UPDATE AD_Column SET FKConstraintName='CreatedBy_MStorageReservationL', FKConstraintType='N',Updated=TO_TIMESTAMP('2021-11-01 16:20:27','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=214600
+;
+
+-- Nov 1, 2021, 4:20:27 PM MYT
+UPDATE AD_Column SET FKConstraintName='MAttributeSetInstance_MStorage', FKConstraintType='N',Updated=TO_TIMESTAMP('2021-11-01 16:20:27','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=214613
+;
+
+-- Nov 1, 2021, 4:20:27 PM MYT
+UPDATE AD_Column SET FKConstraintName='MProduct_MStorageReservationLo', FKConstraintType='N',Updated=TO_TIMESTAMP('2021-11-01 16:20:27','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=214612
+;
+
+-- Nov 1, 2021, 4:20:27 PM MYT
+UPDATE AD_Column SET FKConstraintName='MWarehouse_MStorageReservation', FKConstraintType='N',Updated=TO_TIMESTAMP('2021-11-01 16:20:27','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=214611
```

Comment:
This line is throwing error in postgresql - stopping the process of SyncDB.

Can you please apply the following patch that solves the issue:
[pr961_fix.txt](https://github.com/idempiere/idempiere/files/7466829/pr961_fix.txt)

Regards,

Carlos Ruiz